larndsim/consts/light.py

```diff
@@ -34,5 +83,33 @@ def set_light_properties(detprop_file):
         LUT_VOX_DIV = np.array(detprop['lut_vox_div'])
         N_OP_CHANNEL = detprop['n_op_channel']
         OP_CHANNEL_EFFICIENCY = np.array(detprop['op_channel_efficiency'])
+        
+        tpc_to_op_channel = detprop['tpc_to_op_channel']
+        OP_CHANNEL_TO_TPC = np.zeros((N_OP_CHANNEL,), int)
+        for itpc in range(len(tpc_to_op_channel)):
+            for idet in range(N_OP_CHANNEL):
+                if idet in tpc_to_op_channel[itpc]:
+                    OP_CHANNEL_TO_TPC[idet] = itpc
+
+        LIGHT_TICK_SIZE = float(detprop.get('light_tick_size', LIGHT_TICK_SIZE))
```

Comment:
I see these are now loaded from the YAML, if present, otherwise they assume the default values. However, it would be nice for reference to have these values in the module0 YAML file